Ghidra/Features/GraphServices/src/main/java/ghidra/graph/visualization/DefaultGraphDisplay.java

```diff
@@ -345,17 +345,13 @@ private void connectSelectionStateListeners() {
 				Collection<AttributedVertex> selectedVertices = getVertices(e.getItem());
 				List<String> selectedVertexIds = toVertexIds(selectedVertices);
 				notifySelectionChanged(selectedVertexIds);
-				multiSelectedVertexPaintable.setSelectedVertices(selectedVertices);
```

Comment:
Class no longer maintains selected vertex state. instead uses the SelectedState in the viz model